### ISSUE TITLE
It is still in development?

### ISSUE THREAD

**Issue by Giuseppe-TD** [crater-invoice/crater#1277]
This project is still in development? 
We will have some updates?

**Comment by rihards-simanovics** [comment]
No, please see here: https://github.com/crater-invoice/crater/issues/1265, please kindly close this issue so the [original](https://github.com/crater-invoice/crater/issues/1265) one is still visible.

**Comment by timothyoesch** [comment]
I suppose in that case it'd be great to add this information to the readme?

**Comment by rihards-simanovics** [comment]
https://github.com/crater-invoice/crater/pull/1263

**Comment by timothyoesch** [comment]
Sorry, didn't see that. Thanks a bunch!

**Comment by rihards-simanovics** [comment]
# ❗Attention❗
Please migrate to InvoiceShelf, a fork of crater which continues the development of this project.

More info on new fork availible in this issue: #1281
